lib/spack/spack/ci.py

```diff
@@ -1022,7 +1022,7 @@ def push_mirror_contents(env, spec, yaml_path, mirror_url, build_id):
         tty.debug('Creating buildcache')
         buildcache._createtarball(env, yaml_path, None, True, False,
                                   mirror_url, None, True, False, False, True,
-                                  False)
+                                  True)
```

Comment:
Can we also use keyword args here? This is really hard to read with a sea of arguments.

Reply:
Yes, it's time I finally do that.